projects/common/src/lib/components/iot-gateway-details-charts/iot-gateway-image/iot-gateway-image.component.ts

```diff
@@ -64,9 +69,9 @@ export class CommonIotGatewayImageComponent implements OnInit, OnDestroy, AfterV
 
       this.combinedReadingSelection.clear();
 
-      this.graphService.getReadings(this.device.name, this.instrument.name, 30)
+      this.graphService.getReadings(this.gatewayId,this.device.name, this.instrument.name, 30)
```

Comment:
There's some lint here... plus a lot of the changes seem to be simple spaces or tab conversion. Maybe the linter would clean those up and reduce the change set?

Reply:
The space/tab conversions were inconsistent spacing that the linter didn't pick up. They are generated from 'Format This Document' option in Visual Studio Code.

Reply:
Ok, but shouldn't the linter add a space in here after the comma?

Reply:
It does. I think I did the formatting before, hence why the lint occurs.

Reply:
I did just run an `npm run lint` but it doesn't seem to format those spaces.

Reply:
That sounds odd. It used to do that... didn't it? If it wasn't that, then perhaps the save actions of the ide would do it, but I thought it was the linter's role.

Reply:
The documentation seems to agree with me, but for some reason, it does not appear to make those changes... https://eslint.org/docs/latest/rules/comma-spacing

Reply:
I also looked at upgrading the ESlint dependencies, but one path (full upgrade) resulted in a run time error. 

Then the second path of upgrading only a few components (skipping the eslint-builder which seemed to be the cause of the run time error). WIthout any errors in the `npm run lint` command, this also did not modify any of the files to fix up the comma-spacing we've been talking about.

Reply:
This is now fixed.